include/fi_util.h

```diff
@@ -418,23 +433,26 @@ struct util_cmap {
 
 	struct ofi_key_idx key_idx;
 
+	uint8_t closing;
```

Comment:
Can we expand/use the cmap_state for this, rather than maintaining another state variable?